notebook/static/notebook/js/codecell.js

```diff
@@ -467,7 +469,7 @@ define([
         } else {
             ns = encodeURIComponent(prompt_value);
         }
-        return '<bdi>In</bdi>&nbsp;[' + ns + ']:';
+        return '<bdi>'+i18n._('In')+'</bdi>&nbsp;[' + ns + ']:';
```

Comment:
I suspect that this is going to look messy with anything other than 2 characters. That's not up to you to fix, though.